### DISCUSSION TITLE
Let's hang out at Matrix

### DISCUSSION THREAD

**Discussion by fmartingr** [go-shiori/shiori#633] [Announcements]
Hey all!

When I started on the maintenance of Shiori I opened an IRC channel (#shiori@libera.chat) for people to hang out. Reality is that very few people actually joined and the ephemeral behavior of IRC makes async communication/discussion hard, with people joining for help/questions/contributions but leaving, so my answers could never reach them, and I can't expect everyone to have a bouncer or service to be connected 24/7 so they could check if they received an answer.

That said I've seen the trend of Matrix going along on several other projects so I've gone ahead and created a Shiori space under the matrix.org server at [#shiori-general:matrix.org](https://matrix.to/#/#shiori:matrix.org), please join if you feel like it! I have the client on my desktop and will try to check it out frequently and share updates and ideas along with the community.

Thank you all!

**Comment by LoisGNS** [comment]
Is this operative? I have never used Matrix before and all my attempts to set up an account & connect the server aren't working. If it's still going on and is the main place for questions, I'd appreciate any help in getting started. 

I set up an Element.io account, but can't seem to find the referenced server.

**Reply by fmartingr** [reply]
Yes it is. Can you open this link to join the Shiori space? https://matrix.to/#/#shiori:matrix.org The space and my account are on that same server (matrix.org, which is the one element uses).

**Comment by LoisGNS** [comment]
I've been having difficulty getting in, but may have finally done so, although it seems to time out frequently. I just posted a thread-reply in General and help request in the help/support channel (as archive-me). I hope I'm doing this correctly, as I've never used Matrix before, and have only been using Shiori for a few days (moving links from Pocket). As I work on it, I'm coming with all sorts of questions, and want to know the best place to ask them. :)
